### PR TITLE
Updating Coverage Figure

### LINKED ISSUE #157: Missing x axis label in main coverage figure

### PR DESCRIPTION
**Pull request by gwaygenomics**
Closes #157 - also updating plots.

I also realized I switched the axes of one panel :scream: fixed now

### CODE REVIEW

**Comment by gwaygenomics** [comment]
## Main Figure

main_coverage

## Supplemental Figure

supplemental_coverage

File: 7.analyze-coverage/scripts/nbconverted/visualize_coverage.r [hunk 1]
```diff
@@ -410,8 +410,8 @@ tcga_cm_label <- cowplot::ggdraw() +
                         angle = 90)
 
 label_main <- cowplot::plot_grid(tcga_tf_label,
-                                 tcga_cm_label,
                                  tcga_react_label,
+                                 tcga_cm_label,
```
Comment (ajlee21) [inline 7.analyze-coverage/scripts/nbconverted/visualize_coverage.r]: Not seeing a difference in the figure axes.  What changed?

Comment (gwaygenomics) [reply 7.analyze-coverage/scripts/nbconverted/visualize_coverage.r]: The bottom left x axis was not labeled

**Comment by gwaygenomics** [comment]
> Not sure if you want to link to your workflow image to complement the summary you have at the beginning of the file.

 > Might considering moving the description of the 3 types of plots closer to where it is in the code

Hmmm, i think these are good ideas, but I think the best place to add these descriptions is in the README

**Pull request merged**